[RHDEVDOCS-2332] - Documentation of using the Developer Catalog (#29275)
openshift/openshift-docs: modules/odc-grouping-multiple-components.adoc

.Procedure

. Create and deploy the MariaDB service to your project as follows:
.. In the *Developer* perspective, navigate to the *+Add* view and select the *From Catalog* option to view all of the available services in the *Developer Catalog*.
.. Select *Databases* to list all the database services, and then click on the *MariaDB* option to see the details for the service.
.. Click *Instantiate Template* to see an automatically populated template with details for the MariaDB service, and then click *Create* to create and view the MariaDB service in the *Topology* view.

> . To add the MariaDB service to the existing application group, kbd:[Shift]+ dragging the *mariadb* pod to the application group containing the Node.Js component; the MariaDB service is added to the existing application group.

Reviewer: This sequence doesn't follow the guidance in the IBM Style Guide for keyboard shortcuts. I think that 
```suggestion
. To add the MariaDB service to the existing application group, press kbd:[Shift]+ and drag the *mariadb* pod to the application group that contains the Node.Js component. The MariaDB service is added to the existing application group.
```

> . Click on the MariaDB service node to see the label `app.kubernetes.io/part-of=<application-name>` added to the *Labels* section in the *Overview* Panel. kbd:[Shift]+ dragging a component and adding it to an application group automatically adds the required labels to the component.

Reviewer: ```suggestion
. Click the MariaDB service node and confirm that the  `app.kubernetes.io/part-of=<application-name>`label is in the *Labels* section in the *Overview* Panel. Adding a component to an application group automatically adds the required labels to the component.
```

+
.Application grouping
image::odc_app_grouping_label.png[]

> Alternatively, you can also add a component in the *Topology* view to an application as follows:

Reviewer: This information shouldn't be at the bottom. It should be presented as an alternate set of substeps around line 23.

> . To add the MariaDB service to your application grouping, click on the *mariadb* pod to see the *Overview* panel to the right.

Reviewer: ```suggestion
. To add the MariaDB service to your application grouping, click the *mariadb* pod to see the *Overview* panel to the right.
```

.Procedure

. Create and deploy the MariaDB service to your project as follows:

> .. In the *Developer* perspective, navigate to the *+Add* view and select the *Database* option to see the *Developer Catalog*, which has multiple options that you can add as components or services to your application.

Reviewer 2: ```suggestion
.. In the *Developer* perspective, navigate to the *+Add* page and select the *Database* option to see the *Developer Catalog*, which has multiple options that you can add as components or services to your application.
```

Reviewer 2: When user clicks on Database card on +Add page, it will open the Developer Catalog page with Database options. In above sentence, it seems that you have added description of Developer Catalog page instead it should what user will see when he/she clicks on Database card on Add page. WDYT about updating above sentence? @Preeticp @rishumehra

Reviewer 3: @gajanan-more the change from view to page, if done needs to be uniform across the OCP doc set. If required we will pick that change up through another PR.
Suggest similar wording as we have in the other module:
. In the *Developer* perspective, navigate to the *+Add* page and select the *Database* option to view all the available services in the *Developer Catalog*.
. Select *Databases* to list all the database services, and then click on the MariaDB option to see the details for the service.
Does this make it more clear?

Reviewer 2: Option 1: 
. In the Developer perspective, navigate to the +Add page and select the From Catalog option to view all the available services in the Developer Catalog.
. Select Databases to list all the database services, and then click on the MariaDB option to see the details for the service.

Option 2:
. In the Developer perspective, navigate to the +Add page and select the Database option to view all the available database services present in the Developer Catalog under Database.
. Select MariaDB option to see the details for the service.

WDYT @Preeticp

Reviewer 3: @gajanan-more option 1 seems perfect.

Author: Option 1 seems fine to me also. Will incorporate that in the new commit.

.. Click on the *MariaDB* option to see the details for the service.
.. Click *Instantiate Template* to see an automatically populated template with details for the MariaDB service, and click *Create* to create the service.

> . On the left navigation panel, click *Topology* to see the MariaDB service deployed in your project.

Reviewer 2: User will automatically redirected to Topology page and doesn't need to click anywhere.

Reviewer 2: Yes

> . To add the MariaDB service to the existing application group, select the *mariadb* pod and drag it to the application; the MariaDB service is added to the existing application group.

Reviewer 3: to the application group containing the Node.Js component;

Author: Done, edited the sentence as suggested.

> . Dragging a component and adding it to an application group automatically adds the required labels to the component. Click on the MariaDB service node to see the label `app.kubernetes.io/part-of=myapp` added to the *Labels* section in the *Overview* Panel.

Reviewer 3: s/`app.kubernetes.io/part-of=myapp`/`app.kubernetes.io/part-of=<application-name>`

Author: Will change the application name.

+
.Application grouping
image::odc_app_grouping_label.png[]

> Alternatively, you can also add the component to an application as follows:

Reviewer 3: ...you can also add a component in the *Topology* view to an application...

Author: Done.
